src/test/java/nl/dtls/fairdatapoint/api/controller/MetadataControllerTest.java

```diff
@@ -266,6 +272,27 @@ public void supportedAcceptHeaders() throws Exception {
         assertEquals(HttpServletResponse.SC_OK, response.getStatus());
     }
     
+    @DirtiesContext
+    @Test
+    public void testOncePerRequestFilter() throws Exception {
```

Comment:
I know we've discussed this before, but I can't remember the conclusion. Since this test does not change anything in the context (e.g. no POST calls are made), does this still need `@DirtiesContext`?

Reply:
We use @DirtiesContext because of @Before method in this section (https://github.com/DTL-FAIRData/FAIRDataPoint/blob/develop/src/test/java/nl/dtls/fairdatapoint/api/controller/MetadataControllerTest.java#L95). We have to refractor the @Before method to get rid of @DirtiesContext in the get calls, but the time which we gain out of this refractory approach is not significant, so in the previous discussion we conclude not to do the refractory ;-)